pilot/pkg/networking/core/v1alpha3/envoyfilter/monitoring.go

```diff
@@ -63,30 +62,17 @@ func IncrementEnvoyFilterMetric(name string, pt PatchType, applied bool) {
 	if !features.EnableEnvoyFilterMetrics {
 		return
 	}
-	result := Applied
-	if !applied {
-		result = Skipped
+	// Only set the Gauge when the filter is atleast applied once.
+	if applied {
+		envoyFilterStatus.With(nameType.Value(name)).With(patchType.Value(string(pt))).
```

Comment:
don't you need to set it to 0 when its not applied?

Reply:
So we want to set it to only 1 if is every applied if not it would be zero by default right? That was my assumption.